repository-data/application/src/main/resources/hcm-config/hst/hosts.yaml

```diff
@@ -70,6 +71,7 @@ definitions:
                 jcr:primaryType: hst:virtualhost
               jcr:primaryType: hst:virtualhost
             jcr:primaryType: hst:virtualhost
+          hst:scheme: http
```

Comment:
Is it worth adding a comment: # No SSL in local dev environment for simplicity

Reply:
Can't comment in YAML, see above